bugbot/rules/file_crash_bug.py

```diff
@@ -41,15 +43,48 @@ def get_bz_params(self, date):
             "keywords_type": "allwords",
         }
 
+    def _active_regression_authors(
+        self, signatures: list[SignatureAnalyzer]
+    ) -> set[str]:
+        """Get Bugzilla usernames for users who are active and can be needinfo'd.
+
+        Args:
+            signatures: signatures to check the status if their regression
```

Comment:
```suggestion
            signatures: a list of signatures for which to check the status of their regression author
```

Reply:
Done in 65ae5c269658bfdb2a9c28800c43a6a03fb5c425.